xc7/utils/prjxray_edge_library.py

```diff
@@ -688,6 +688,26 @@ def connect_at(
                     yield (src_node, switch_pkey, dest_node, pip.pip_pkey)
                     return
 
+        elif self.pins and other_connector.pins and pip.is_pseudo:
+            for pin_dir in self.pins.edge_map:
+                if pin_dir in other_connector.pins.edge_map:
+                    src_node = self.pins.edge_map[pin_dir]
+                    dest_node = other_connector.pins.edge_map[pin_dir]
+
+                    src_wire_switch_pkey, src_wire_node = self.find_wire_node(
+                        src_wire_pkey, src_node, None
+                    )
+
+                    dest_wire_switch_pkey, dest_wire_node = self.find_wire_node(
+                        dest_wire_pkey, dest_node, None
+                    )
+
+                    yield (
+                        src_wire_node, src_wire_switch_pkey, dest_wire_node,
```

Comment:
Do not use src_wire_switch_pkey, use the switch belonging to the pip itself.  Should be able to use 
```
                    switch_pkey = pip.get_pip_switch(
                        src_wire_pkey, dest_wire_pkey
                    )
``` or equiv.

Reply:
Done